src/node/node_state.h

```diff
@@ -1724,18 +1724,15 @@ namespace ccf
                 network.secrets.get_name()));
             }
 
-            auto encrypted_ledger_secrets = w.at(0);
-            if (!encrypted_ledger_secrets.has_value())
+            auto secrets_for_nodes = w.at(0);
```

Comment:
Keeping "ledger" in the name would be good, imo, we have several different kind of secrets.